### PR TITLE
docs: cluster profile resournce , multiple packs example

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR addresses an issue encountered by users when adding Packs to the cluster profile resource. 

CleanShot 2022-10-31 at 09 54 56

### CODE REVIEW

File: docs/resources/cluster_profile.md [hunk 1]
```diff
@@ -142,6 +144,20 @@ resource "spectrocloud_cluster_profile" "profile" {
 }
 ```
 
+### Example of Providing Multiple Packs
+
+You can provide multiple packs at once by leveraging a dynamic block.  
+
+!> The order of the Packs must be taken into consideration so avoid any situations where the order of the Packs is re-arranged by Terraform, such as nested loops `{ for pack in var.packs : pack => pack }`. We recommend creating a variable that contains the list of Packs arranged in order of the App Profile stack and their respective configuration.
```
Comment (kreeuwijk) [inline docs/resources/cluster_profile.md:151]: Can you make the example `for_each = { for pack in var.packs : pack => pack }`

**Review by kreeuwijk — CHANGES_REQUESTED** [review]

File: docs/resources/cluster_profile.md (hunk 1 above)
Comment (karl-cardenas-coding) [reply docs/resources/cluster_profile.md:151]: @kreeuwijk I'm not sure I am following. Are you wanting the code snippet to have the following?
```terraform
resource "spectrocloud_cluster_profile" "this" {
  name = "security-profile"
  dynamic "pack" {
    for_each = { for pack in var.packs : pack => pack }
  }
```

Comment (kreeuwijk) [reply docs/resources/cluster_profile.md:151]: No I mean changing

"...such as nested loops `{ for pack in var.packs : pack => pack }`. We recommend..."
to
"...such as nested loops `for_each = { for pack in var.packs : pack => pack }`. We recommend..."

Comment (karl-cardenas-coding) [reply docs/resources/cluster_profile.md]: Fixed ✅

**Pull request merged**